src/lazy/text/raw/reader.rs

```diff
@@ -155,6 +157,15 @@ mod tests {
             3
         ]
         
+        {
+            // Identifier 
+            foo: 100,
+            // Quoted symbol
+            'bar': 200,
+            // Short-form string
+            "baz": 300
```

Comment:
Would be good to have some comments in unexpected (but legal) locations.
```suggestion
            "baz": 300,
            /* Comments... */ qux /* comments */ : /* are */ 400 /* everywhere */,
```